axelor-bank-payment/src/main/java/com/axelor/apps/bankpayment/service/bankreconciliation/BankReconciliationService.java

```diff
@@ -523,25 +523,33 @@ protected BigDecimal computeStatementUnreconciledLineBalance(
   protected BigDecimal computeStatementOngoingReconciledLineBalance(
       BigDecimal statementOngoingReconciledBalance, BankReconciliationLine brl) {
     if (!brl.getIsPosted() && !Strings.isNullOrEmpty(brl.getPostedNbr())) {
-      statementOngoingReconciledBalance =
-          statementOngoingReconciledBalance.subtract(brl.getDebit());
-      statementOngoingReconciledBalance = statementOngoingReconciledBalance.add(brl.getCredit());
+      String query = "self.moveLine = %s";
+      query = query.replace("%s", brl.getMoveLine().getId().toString());
+      List<BankReconciliationLine> bankReconciliationLines =
+          bankReconciliationLineRepository.all().filter(query).fetch();
```

Comment:
You can just write:
bankReconciliationLineRepository.all().filter("self.moveLine.id = ?1", brl.getMoveLine().getId())

And it's strange that "self.moveLine = id" doesn't throw an exception